.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
I think we should remove it because we don't expect the text editor/IDE of anyone who clone this project. There shouldn't be any `.atom`, `.vscode`, `nbproject`, `.metadata`, `.idea`... I would not like myself having to delete those folders if I use, for example, **vim**, Brackets,Sublime Text, etc. Any thoughts about this?